src/integrationTests/callbacks.test.ts

```diff
@@ -131,29 +128,25 @@ testCookieAndLocalStorage(() => {
           identity,
         });
       });
-      // this test is only passing for cookies
-      test('it should receive subsequent identity updates', async () => {
+
+      test('it should receive subsequent identity updates', (done) => {
```

Comment:
This is my first time seeing `done` - could you please help me understand why we need it here? I can see it improves the coverage of the test.

Reply:
`done` is used only when you want to test a callback, Jest will wait until the done callback is called before finishing the test.
https://jestjs.io/docs/asynchronous#callbacks